CHANGELOG.md

```diff
@@ -9,6 +9,8 @@
 - Decrease memory usage by not storing genesis blocks. This has the effect that
   the database produced by node versions >= 4.2.* cannot be used by node
   versions <= 4.1. The other direction works.
+- Fix issue #375 which resulted in a wrong rpc response when the sender of a 
```

Comment:
The changelog should not have issue numbers. Please explain what it gives to the user in plain terms.